test/packages/PkgUsingJET.jl/test/runtests.jl

```diff
@@ -0,0 +1,4 @@
+using ReTestItems, PkgUsingJET
+
+runtests(PkgUsingJET; verbose_results=true, nworkers=1)
+runtests(PkgUsingJET; verbose_results=true, nworkers=1, worker_init_expr=quote import JET end)
```

Comment:
This is to illustrate that for package extensions to kick in, the weak dependency must be *loaded*. Here is how the two invocations looks:
![image](https://github.com/JuliaTesting/ReTestItems.jl/assets/2074313/41dff24f-9e4a-4ea0-a6dd-eba76dd1ce51)

The first one doesn't run with JET, because on the workers, JET is not loaded.